webpack/scenes/BootedContainerImages/BootedContainerImagesPage.js

```diff
@@ -0,0 +1,242 @@
+                  >
+                    {keysToColumnNames[k]}
+                  </Th>
+                ))}
+              </>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {status === STATUS.PENDING && results.length === 0 && (
+              <Tr ouiaId="table-loading">
+                <Td colSpan={100}>
+                  <EmptyPage
+                    message={{
+                      type: 'loading',
+                      text: __('Loading...'),
+                    }}
+                  />
+                </Td>
+              </Tr>
+            )}
+            {!status === STATUS.PENDING &&
+              results.length === 0 &&
+              !errorMessage && (
+                <Tr ouiaId="table-empty">
+                  <Td colSpan={100}>
+                    <EmptyPage
+                      message={{
+                        type: 'empty',
+                      }}
+                    />
+                  </Td>
+                </Tr>
+            )}
+            {errorMessage && (
+              <Tr ouiaId="table-error">
+                <Td colSpan={100}>
+                  <EmptyPage message={{ type: 'error', text: errorMessage }} />
+                </Td>
+              </Tr>
+            )}
+          </Tbody>
+          {results?.map((result, rowIndex) => {
+            const { bootcBootedImage, digests } = result;
+            const isExpanded = imageIsExpanded(bootcBootedImage);
+            return (
+              <Tbody key="bootable-container-images-body" isExpanded={isExpanded}>
+                <Tr key={bootcBootedImage} ouiaId={`table-row-${rowIndex}`}>
+                  <>
+                    <Td
+                      expand={digests.length > 0 && {
+                        rowIndex,
+                        isExpanded,
+                        onToggle: (_event, _rInx, isOpen) =>
+                          expandedImages.onToggle(isOpen, bootcBootedImage),
+                        expandId: 'booted-containers-expander',
+                      }}
+                    />
+                    {columnNamesKeys.map(k => (
+                      <Td
+                        key={k}
+                        dataLabel={keysToColumnNames[k]}
+                      >
+                        {columns[k].wrapper ? columns[k].wrapper(result) : result[k]}
+                      </Td>
+                    ))}
+                  </>
+                </Tr>
+                {digests ?
+                  <Tr isExpanded={isExpanded} ouiaId={`table-row-outer-expandable-${rowIndex}`}>
+                    <Td />
+                    <Td colSpan={3}>
+                      <ExpandableRowContent>
+                        <TableComposable variant="compact" isStriped ouiaId={`table-composable-expanded-${rowIndex}`}>
+                          <Thead>
+                            <Tr ouiaId={`table-row-inner-expandable-${rowIndex}`}>
+                              <Th width={50}>{__('Image digest')}</Th>
```

Comment:
I tried to use the width to align the inner "Hosts" data with the outer "Hosts" data, but they are always off by a little bit. So, for now, I distanced them enough to not look completely weird. Ideally I could remove the Hosts header in the inner expanded table and have that data be perfectly lined up with the outer table's hosts count. However, I'm not sure if that's easily possible with these PF components.

Reply:
Hard-coding width or height in pixels is usually not advisable, because everything can change depending on the user's text size in the browser. :thinking:

Reply:
The width there that I'm setting is a percentage, so it should adapt.